services/QuillLMS/db/migrate/20240318155131_add_name_to_evidence_research_gen_ai_passage.evidence.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# This migration comes from evidence (originally 20240318155008)
```

Comment:
Good question. I closed that config change [PR](https://github.com/empirical-org/Empirical-Core/pull/11565#issuecomment-1979602826) a couple weeks ago since I hit a weird error (which I think I have resolved now)  

I've been debating re-opening that PR now given the utility with all of these PRs.  But, I wanted to get the ball rolling on these experiments first.